docs/graphql/manual/queries/pagination.rst

```diff
@@ -188,3 +188,53 @@ Limit results in a nested object
         ]
       }
     }
+
+Fetch aggregate count with limited results
+------------------------------------------
+
+``count`` (as part of aggregations) can be used to do pagination.
+
+**Example:** Fetch a list of articles where a certain condition is true and get their count. Then limit the number of articles to display.
+
+.. graphiql::
+  :view_only:
+  :query:
+    query articles ($where: articles_bool_exp!) {
+      articles_aggregate(where: $where) {
+        aggregate {
+          totalCount: count
+        }
+      }
+      articles (where: $where limit: 10 order_by: {rating: desc}) {
```

Comment:
```suggestion
      articles (where: $where limit: 4 order_by: {rating: desc}) {
```

Reply:
This because you're only getting 4 results in the response, which doesn't add up to the total and limit values.